Lib/typing.py

```diff
@@ -1030,44 +1051,98 @@ def __ror__(self, left):
         return Union[left, self]
 
     @_tp_cache
-    def __getitem__(self, params):
+    def __getitem__(self, args):
+        """Parameterizes an already-parameterized object.
+
+        For example, we arrive here doing something like:
+          T1 = TypeVar('T1')
+          T2 = TypeVar('T2')
+          T3 = TypeVar('T3')
```

Comment:
Extra TypeVar here.

Reply:
Oops! I meant the last line to be `C = B[T3]` so that readers don't feel uneasy re-using the same type variable for both `B` and `C`. I've fixed that line and kept this `TypeVar`.